style.css

```diff
@@ -0,0 +1,43 @@
+.modal {
+  display: none;
```

Comment:
I think bootstraps sets this default to none already - no need to style it as such here.

https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_modals.scss#L17